qtpy/tests/test_main.py

```diff
@@ -1,6 +1,10 @@
 import os
+import sys
+import subprocess
 
-from qtpy import QtCore, QtGui, QtWidgets
+import pytest
+
+from qtpy import QtCore, QtGui, QtWidgets, API_NAMES, PythonQtValueError
 try:
```

Comment:
```suggestion

try:
```

Adding a blank line before this one for clarity.